sdk/ml/azure-ai-ml/azure/ai/ml/entities/_load_functions.py

```diff
@@ -742,6 +744,36 @@ def load_connection(
     return cast(Connection, load_common(Connection, source, relative_origin, **kwargs))
 
 
+# Unlike other aspects of connections, this wasn't made experimental, and thus couldn't just be replaced
+# During the renaming from 'workspace connection' to just 'connection'.
+def load_workspace_connection(
```

Comment:
as a follow up item, can we add a deprecation message? The first question we'll get is probably "when do I use `load_connection` vs `load_workspace_connection`", so we should make it clear we prefer users to just call `load_connection` instead.